infra/docker/Dockerfile

```
@@ -0,0 +1,32 @@
+WORKDIR /app
+
+COPY go.* ./
+RUN go mod download
+
+COPY . .
+RUN go build -o main .
+
+FROM alpine:3.19
+
+# Install ca-certificates for HTTPS requests
+RUN apk --no-cache add ca-certificates
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)